.github/workflows/test.yml

```diff
@@ -44,22 +44,20 @@ jobs:
       with:
         python-version: 3.8
     
-    - name: Activate python cache
-      uses: actions/cache@v3
-      with:
-        path: |
-          ${{ env.pythonLocation }}
-          ${{ env.HF_HOME }}
-        key: ${{ env.pythonLocation }}-${{ matrix.pytorch-version }}-${{ matrix.test-kind }}-${{ hashFiles('setup.py') }}
+    - name: Setup uv
+      run: | 
+          curl -LsSf https://astral.sh/uv/install.sh | sh
```

Comment:
This script seems to always point to the latest release, so there is no way to control the version with this install method. Might not be an issue, we'll probably have to see how stable uv turns out to be.